2017-09-26 22:02:01,238 [main] DEBUG org.jboss.logging - Logging Provider: org.jboss.logging.Slf4jLoggerProvider 
2017-09-26 22:32:38,631 [nREPL-worker-14] ERROR db-devops.db.core - /Users/chenjinfen/Desktop/projects/CMB/IMAVP/2017/db-devops/05-dev/db-devops/user.edn (No such file or directory) 
2017-09-26 22:32:38,673 [nREPL-worker-14] ERROR db-devops.db.core - /Users/chenjinfen/Desktop/projects/CMB/IMAVP/2017/db-devops/05-dev/db-devops/verification.edn (No such file or directory) 
2017-09-26 22:32:38,677 [nREPL-worker-14] INFO  db-devops.env - 
-=[db_devops started successfully using the development profile]=- 
2017-09-26 22:32:38,972 [nREPL-worker-14] INFO  luminus.http-server - starting HTTP server on port 3000 
2017-09-26 22:32:39,099 [nREPL-worker-14] INFO  org.xnio - XNIO version 3.3.6.Final 
2017-09-26 22:32:39,381 [nREPL-worker-14] INFO  org.projectodd.wunderboss.web.Web - Registered web context / 
2017-09-26 22:33:24,892 [XNIO-1 task-10] INFO  db-devops.routes.services.auth - user: admin successfully logged in from 0:0:0:0:0:0:0:1 localhost 
2017-09-26 22:33:57,856 [XNIO-1 task-5] DEBUG db-devops.upgrade-api-client - request sent to upgrade API:  {:basic-auth [ooflow ooflow], :content-type :json, :body {"inputParams":{"ItilID":"asf","USER":false,"NODENUM":1,"FS":false,"DB":true,"CHECK":"YES","DISKLIST":"asdf","DBM":true,"DB2VER":"V10.5.5","USERINFO":"asdf","CODEPAGE":"1386","INITDB2PATH":"asdf","DBNAME":"asdf","SYS":true,"DB2PATH":"asdf","DBDATA":"1","HOSTNAME":"asdf","INST":false},"user_id":"admin","user_name":"admin","funcid":"CreateDB","label":"步骤1：创建数据库","shellNames":[{"shellID":"caller1","shellParams":[{"value":"inputParams.HOSTNAME"},{"value":"root"},{"value":"inputParams.INITDB2PATH"},{"name":"-DBDATASIZEGB","value":"inputParams.DBDATA"},{"name":"-USERINFO","value":"inputParams.USERINFO"},{"name":"-DB","value":"inputParams.DBNAME"},{"name":"-DB2PATH","value":"inputParams.DB2PATH"},{"name":"-DISKLIST","value":"inputParams.DISKLIST"},{"name":"-itil","value":"inputParams.itil"},{"name":"-CODEPAGE","value":"inputParams.CODEPAGE"},{"name":"-NODENUM","value":"inputParams.NODENUM"},{"name":"-DB2VER","value":"inputParams.DB2VER"},{"name":"-CHECK","value":"inputParams.CHECK"}]}]}, :method :post, :url http://localhost:8080/rest/operationService/submitOperation} 
2017-09-26 22:33:58,107 [XNIO-1 task-5] ERROR db-devops.conditionhandling - Exception occured during calling function: clj_http.client$wrap_unknown_host$fn__41694@529da22a ({:basic-auth ["ooflow" "ooflow"], :content-type :json, :body "{\"inputParams\":{\"ItilID\":\"asf\",\"USER\":false,\"NODENUM\":1,\"FS\":false,\"DB\":true,\"CHECK\":\"YES\",\"DISKLIST\":\"asdf\",\"DBM\":true,\"DB2VER\":\"V10.5.5\",\"USERINFO\":\"asdf\",\"CODEPAGE\":\"1386\",\"INITDB2PATH\":\"asdf\",\"DBNAME\":\"asdf\",\"SYS\":true,\"DB2PATH\":\"asdf\",\"DBDATA\":\"1\",\"HOSTNAME\":\"asdf\",\"INST\":false},\"user_id\":\"admin\",\"user_name\":\"admin\",\"funcid\":\"CreateDB\",\"label\":\"步骤1：创建数据库\",\"shellNames\":[{\"shellID\":\"caller1\",\"shellParams\":[{\"value\":\"inputParams.HOSTNAME\"},{\"value\":\"root\"},{\"value\":\"inputParams.INITDB2PATH\"},{\"name\":\"-DBDATASIZEGB\",\"value\":\"inputParams.DBDATA\"},{\"name\":\"-USERINFO\",\"value\":\"inputParams.USERINFO\"},{\"name\":\"-DB\",\"value\":\"inputParams.DBNAME\"},{\"name\":\"-DB2PATH\",\"value\":\"inputParams.DB2PATH\"},{\"name\":\"-DISKLIST\",\"value\":\"inputParams.DISKLIST\"},{\"name\":\"-itil\",\"value\":\"inputParams.itil\"},{\"name\":\"-CODEPAGE\",\"value\":\"inputParams.CODEPAGE\"},{\"name\":\"-NODENUM\",\"value\":\"inputParams.NODENUM\"},{\"name\":\"-DB2VER\",\"value\":\"inputParams.DB2VER\"},{\"name\":\"-CHECK\",\"value\":\"inputParams.CHECK\"}]}]}", :method :post, :url "http://localhost:8080/rest/operationService/submitOperation"}). Original exception message: java.net.ConnectException: Connection refused (Connection refused) 
2017-09-26 22:35:37,214 [XNIO-1 task-18] DEBUG db-devops.upgrade-api-client - request sent to upgrade API:  {:basic-auth [ooflow ooflow], :content-type :json, :body {"inputParams":[{"SRCHOSTIP":"1.1.1.1","TGPORT":"50000","TGDBNAME":"asdf","SRCINSTANCENAME":"asdf","TGHOSTNAME":"asdfsadf","SRCHOSTNAME":"asdf","SRCPORT":"12121","TGHOSTIP":"1.1.1.1","TGINSTANCENAME":"asdfs","SRCDBNAME":"asdf"}],"inputParam":{"POSTTASKNAME":"asdf"},"user_id":"admin","user_name":"admin","funcid":"CreateTSK","label":"步骤2：制定升级任务"}, :method :post, :url http://localhost:8080/rest/operationService/submitOperation} 
2017-09-26 22:35:37,223 [XNIO-1 task-18] ERROR db-devops.conditionhandling - Exception occured during calling function: clj_http.client$wrap_unknown_host$fn__41694@529da22a ({:basic-auth ["ooflow" "ooflow"], :content-type :json, :body "{\"inputParams\":[{\"SRCHOSTIP\":\"1.1.1.1\",\"TGPORT\":\"50000\",\"TGDBNAME\":\"asdf\",\"SRCINSTANCENAME\":\"asdf\",\"TGHOSTNAME\":\"asdfsadf\",\"SRCHOSTNAME\":\"asdf\",\"SRCPORT\":\"12121\",\"TGHOSTIP\":\"1.1.1.1\",\"TGINSTANCENAME\":\"asdfs\",\"SRCDBNAME\":\"asdf\"}],\"inputParam\":{\"POSTTASKNAME\":\"asdf\"},\"user_id\":\"admin\",\"user_name\":\"admin\",\"funcid\":\"CreateTSK\",\"label\":\"步骤2：制定升级任务\"}", :method :post, :url "http://localhost:8080/rest/operationService/submitOperation"}). Original exception message: java.net.ConnectException: Connection refused (Connection refused) 
2017-09-26 22:36:19,179 [XNIO-1 task-17] DEBUG db-devops.upgrade-api-client - request sent to upgrade API:  {:basic-auth [ooflow ooflow], :content-type :json, :body {"inputParams":[{"SRCHOSTIP":"1.1.1.1","TGPORT":"50000","TGDBNAME":"asdf","SRCINSTANCENAME":"asdf","TGHOSTNAME":"asdfsadf","SRCHOSTNAME":"asdf","SRCPORT":"12121","TGHOSTIP":"1.1.1.1","TGINSTANCENAME":"asdfs","SRCDBNAME":"asdf"}],"inputParam":{"POSTTASKNAME":"asdf"},"user_id":"admin","user_name":"admin","funcid":"CreateTSK","label":"步骤2：制定升级任务"}, :method :post, :url http://localhost:8080/rest/operationService/submitOperation} 
2017-09-26 22:36:19,186 [XNIO-1 task-17] ERROR db-devops.conditionhandling - Exception occured during calling function: clj_http.client$wrap_unknown_host$fn__41694@529da22a ({:basic-auth ["ooflow" "ooflow"], :content-type :json, :body "{\"inputParams\":[{\"SRCHOSTIP\":\"1.1.1.1\",\"TGPORT\":\"50000\",\"TGDBNAME\":\"asdf\",\"SRCINSTANCENAME\":\"asdf\",\"TGHOSTNAME\":\"asdfsadf\",\"SRCHOSTNAME\":\"asdf\",\"SRCPORT\":\"12121\",\"TGHOSTIP\":\"1.1.1.1\",\"TGINSTANCENAME\":\"asdfs\",\"SRCDBNAME\":\"asdf\"}],\"inputParam\":{\"POSTTASKNAME\":\"asdf\"},\"user_id\":\"admin\",\"user_name\":\"admin\",\"funcid\":\"CreateTSK\",\"label\":\"步骤2：制定升级任务\"}", :method :post, :url "http://localhost:8080/rest/operationService/submitOperation"}). Original exception message: java.net.ConnectException: Connection refused (Connection refused) 
2017-09-26 22:37:09,262 [XNIO-1 task-7] DEBUG db-devops.upgrade-api-client - {:funcid :CheckDB, :user_id admin} 
2017-09-26 22:37:09,269 [XNIO-1 task-7] DEBUG db-devops.upgrade-api-client - request sent to upgrade API:  {:basic-auth [ooflow ooflow], :content-type :json, :method :get, :url http://localhost:8080/rest/operationService/getTasks/CheckDB, :query-params {:user_id admin}} 
2017-09-26 22:37:09,310 [XNIO-1 task-7] ERROR db-devops.conditionhandling - Exception occured during calling function: clj_http.client$wrap_unknown_host$fn__41694@529da22a ({:basic-auth ["ooflow" "ooflow"], :content-type :json, :method :get, :url "http://localhost:8080/rest/operationService/getTasks/CheckDB", :query-params {:user_id "admin"}}). Original exception message: java.net.ConnectException: Connection refused (Connection refused) 
2017-09-26 22:37:31,919 [nREPL-worker-15] DEBUG org.projectodd.wunderboss.WunderBoss - Returning existing component for org.projectodd.wunderboss.web.Web:-1645144552, ignoring options. 
2017-09-26 22:37:31,933 [nREPL-worker-15] INFO  luminus.http-server - HTTP server stopped 
2017-09-26 22:37:31,937 [nREPL-worker-15] INFO  db-devops.env - 
-=[db_devops has shut down successfully]=- 
